client/components/common/Meta/Switch.vue

```diff
@@ -30,17 +24,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-$size: 15px;
-$checked: #337ab7;
-$unchecked: #949494;
-$lever-checked: lighten($checked, 25%);
-$lever-unchecked: lighten($unchecked, 25%);
-
 .control {
   padding: 3px 8px;
 
-  &:hover {
-    background-color: #f5f5f5;
+  /deep/ label.v-label {
+    margin-bottom: 0;
```

Comment:
I don't think this has any effect on the styling since I can not find a `label` in the `v-switch` component.

Reply:
`label` section of the checkbox/switch components is used to display the optional `description` parameter of the meta component. When added it looks something like this:
<img width="326" alt="Screen Shot 2019-07-30 at 20 35 56" src="https://user-images.githubusercontent.com/20609558/62155762-05364b80-b30a-11e9-84b2-a2c196350689.png">
Without the added css, the label is not perfectly aligned with the checkbox/switch.